Fix compilation problems by upgrading versions

manolo opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gwtproject/gwt-site/223)
<!-- Reviewable:end -->

manolo commented:
Ping @tbroyer

branflake2267 commented:
nice job

tbroyer commented on package.json:
```diff
@@ -1,13 +1,14 @@
-    "grunt-contrib-watch": "~0.5.3",
-    "grunt-contrib-less": "~0.11.4"
+  "devDependencies": {},
+  "dependencies": {
```
Any specific reason for moving those from `devDependencies` to `dependencies`?

manolo commented:
Review status: 0 of 3 files reviewed at latest revision, 1 unresolved discussion.

---

*[package.json, line 5 at r1](https://reviewable.io:443/reviews/gwtproject/gwt-site/223#-Ke3pyJ81Zx92F9kbAkn-r1-5:-Ke9Z4x8R_fAwDw29piM:bhobas) ([raw file](https://github.com/gwtproject/gwt-site/blob/eb28660f2a7485e9f7a03dad72742b353453a2ac/package.json#L5)):*
<details><summary><i>Previously, tbroyer (Thomas Broyer) wrote…</i></summary><blockquote>

Any specific reason for moving those from `devDependencies` to `dependencies`?
</blockquote></details>

No reason, I just removed all dependencies and used `npm --save` to compute latest versions, it updated the dependencies block. I could have used `--save-dev` instead.

Moved to devDependencies now.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/gwtproject/gwt-site/223)*
<!-- Sent from Reviewable.io -->
